posthog/clickhouse/migrations/0172_sessions_v3_recordings_mvs.py

```diff
@@ -0,0 +1,7 @@
+from posthog.clickhouse.client.connection import NodeRole
+from posthog.clickhouse.client.migration_tools import run_sql_with_exceptions
+from posthog.models.raw_sessions.sessions_v3 import RAW_SESSIONS_TABLE_MV_RECORDINGS_SQL_V3
+
+operations = [
+    run_sql_with_exceptions(RAW_SESSIONS_TABLE_MV_RECORDINGS_SQL_V3(), node_roles=[NodeRole.DATA], sharded=True),
```

Comment:
Does this need `sharded=True` ? The source table is `sharded_session_replay_events` and the target table is `writable_raw_sessions_v3` (which is Distributed).